Gothic-Unity-Project/Gothic-Unity
# Feature/29 hvr fix in game menu usage

MainMenu is now usable within normal game.


## Test
* Start game and hit "Menu" button on Quest2 build (not via SteamLink, where this button will open SteamLink menu)
    * (With HVRSimulator, hit _Escape_ like in original G1 to open the menu)
* Alter a value for "Movement" settings.
* Check if this value change is immediately applied to the VR Player

![image](https://github.com/user-attachments/assets/48a50349-bcf2-4260-b1fe-07bb8237bd27)

## Boundaries
* The menu is always in front of us with no smooth movement
* "New Game" isn't working
